rend3-routine/shaders/src/cull.wgsl

```diff
@@ -8,14 +8,120 @@ var<storage> object_buffer: array<Object>;
 @group(0) @binding(2)
 var<storage> culling_job: BatchData;
 @group(0) @binding(3)
-var<storage, read_write> output_buffer: array<u32>;
+var<storage, read_write> primary_draw_calls: array<IndirectCall>;
```

Comment:
We should universally change the primary/secondary verbiage to something better. before/after maybe?